platform/ios/MGLMapboxEvents.m

```diff
@@ -483,22 +565,13 @@ - (void) startTimer {
 
 // Can be called from any thread.
 //
-- (NSString *) getUserAgent {
-    if (self.appName != nil && self.appVersion != nil && self.appBuildNumber != nil && ([self.userAgent rangeOfString:self.appName].location == NSNotFound)) {
-        self.userAgent = [NSString stringWithFormat:@"%@/%@/%@ %@", self.appName, self.appVersion, self.appBuildNumber, self.userAgent];
-    }
-    return self.userAgent;
-}
-
-// Can be called from any thread.
-//
-- (NSString *) formatDate:(NSDate *)date {
-    return [self.rfc3339DateFormatter stringFromDate:date];
+- (NSString *) userAgent {
+    return [NSString stringWithFormat:@"%@/%@/%@ %@", self.appName, self.appVersion, self.appBuildNumber, MGLMapboxEventsUserAgent];
```

Comment:
Do we lose some sort of fallback ability here for when the values were `nil` in the old code?